apps/examples/log_dump/log_dump_main.c

```diff
@@ -61,7 +61,7 @@ int log_dump_main(int argc, char *argv[])
 	printf("This Log Should NOT be saved!!!\n");
 	sleep(1);
 
-	lldbg("\n*********************   LOG DUMP START  *********************\n");
+	printf("\n*********************   LOG DUMP START  *********************\n");
```

Comment:
if (STOP_LOGDUMP_SAVE(fd) < 0) {
		printf("Failed to stop log dump, errno %d\n", get_errno());
		goto errout;
	}

do we have to use lldbg in this failure to STOP case?